factory/factory.Dockerfile

```diff
@@ -55,7 +55,9 @@ RUN ls -la /root \
     # build only dependancies: removed in onbuild step
     bzip2 \
     gnupg \
-    dirmngr
+    dirmngr \
+    openssl \
+    ca-certificates
```

Comment:
move these to be above the `# build only dependancies: removed in onbuild step` comment since these aren't removed in the on build. put a comment above them as for why they're needed.